iron-lazy-pages.html

```diff
@@ -238,15 +247,15 @@
         BEGIN DOM-IF LOGIC
         ##################
        */
-      _loadAndShow: function(cb) {
+      _loadAndShow: function(cb, async) {
```

Comment:
Since `async` is actually a keyword (and GitHub syntax highlighting reminded me about this) this name should also be `loadAsync`.